examples/nemo/eg3/nemo_kernels_trans.py

```diff
@@ -0,0 +1,207 @@
+# -----------------------------------------------------------------------------
```

Comment:
You could drop the "nemo" part from the name of this file as it is in a nemo subdirectory (and other scripts do not specify the api).

Reply:
Done.